DataCollector/PrettyDataCollector.php

```diff
@@ -102,10 +102,10 @@ public function collect(Request $request, Response $response, \Exception $except
 
                         $options = [];
                         if (isset($log['limit'])) {
-                            $options['limit'] = $log['limit']['limitNum'];
+                            $options['limit'] = is_array($log['limit']) ? $log['limit']['limitNum'] : $log['limit'];
                         }
                         if (isset($log['skip'])) {
-                            $options['skip'] = $log['skip']['limitSkip'];
+                            $options['skip'] = is_array($log['skip']) ? $log['skip']['limitSkip'] : $log['limit'];
```

Comment:
Should it maybe be 
```suggestion
                            $options['skip'] = is_array($log['skip']) ? $log['skip']['limitSkip'] : $log['skip'];
```
?

I have no idea :sweat_smile:

Reply:
🤦‍♂